docs/implementations/sci-o.md

```diff
@@ -1,30 +1,29 @@
-# Software Carbon Intensity - Operational Emissions (SCI-O)
+# SCI-O (operational emissions)
 
-Software systems cause emissions through the hardware that they operate on, both through the energy that the physical
-hardware consumes and the emissions associated with manufacturing the hardware. This specification defines a methodology
-for calculating the rate of carbon emissions for a software system. The purpose is to help users and developers make
-informed choices about which tools, approaches, architectures, and services they use in the future. It is a score rather
-than a total; lower numbers are better than higher numbers, and reaching 0 is impossible. This specification is focused
-on helping users and developers understand how to improve software to reduce or avoid the creation of
-emissions. [Read more...](https://github.com/Green-Software-Foundation/sci/blob/main/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md)
+Operational emissions refer to the carbon generated by a component while it is in use. It is the product of the energy used by the component in kWh and the grid intensity in gCO2e/kWh. The operational emissions are added to the embodied emissions (calculated using `sci-m`) to provide an overall SCI score for the component.
 
-## Scope
+## Calculation
 
-To calculate the operational emissions O for a software application, use the following:
+To calculate the operational emissions `o` for a software application, use the following:
 
-O = (E * I)
+```
+o = (energy * grid-ci)
```

Comment:
The SCI specification mentions O , E, I as the variables. I think it will be better to keep them as is.

Reply:
Fair enough, but this is consistent with the model implementation, and I find longer names easier to understand compared to remembering abbreviations. Keep both?

Reply:
Keep both is better I think. SCI specification mentions or so, Therefore we can say equivalent terms in the implementation is this.